### ISSUE TITLE
custom size for window and buttons

### ISSUE THREAD

**Issue by jonpodtu** [dvx/lofi#165]
Hey!

Great app. Was looking for an miniplayer alternative for my ASUS Screenpad, and it works really well. However, for a feature I would like that it was possible to resize the buttons or the window even more than already possible. 

![IMG_20211110_231337](https://user-images.githubusercontent.com/57094931/141203221-54886cff-4673-4986-884f-5e4fd6a78184.jpg)

**Comment by dominik3131** [comment]
Agree, it also would be great if there was minimized mode. For example just simple bar with control buttons. It would be great utility because it could work as toolbar on the task bar in Windows for example.

**Comment by Beast0999** [comment]
> Hey!
> 
> Great app. Was looking for aa miniplayer alternative for my ASUS Screenpad Spotify, and yours works really well. However, for a feature, I would like that it was possible to resize the buttons or the window even more than already possible.
> 
> ![IMG_20211110_231337](https://user-images.githubusercontent.com/57094931/141203221-54886cff-4673-4986-884f-5e4fd6a78184.jpg)

Aye man sick music taste, Mac DeMarco slaps

**Comment by stamoun** [comment]
You'd like the cover window to be bigger? or the playback buttons to be bigger?

I've seen a few requests stating that the layout could be optimized, I'm gonna try and figure what sort of display modes make sense.

From what I can gather it'd be:
- As is (control only visible on hover)
- No cover art only artist/song info
- Only playback buttons
- ... anything else?

**Comment by dominik3131** [comment]
For me simple bar that has only playback buttons and like button would be enough. Something similar is avaiilable in music player called MusicBee, and you can see it here. http://i.imgur.com/B9Aar11.png
